Client/Client.Qt/Widgets/MessageWidget.cpp

```diff
@@ -217,58 +125,48 @@ void MessageWidget::reactionChange(int index)
     }
 }
 
-void MessageWidget::setThisItem(QListWidgetItem* Item) { messageItem = Item; }
-
-void MessageWidget::setMessageText(QString newMessage)
-{
-    if (newMessage != "")
-    {
-        messageText = newMessage;
-        updateWidget();
-    }
-}
-
-void MessageWidget::setStdMessageText(std::string newMessage)
+void MessageWidget::setMessageText(const QString& newMessage)
 {
     if (newMessage != "")
     {
-        messageText = QString::fromStdString(newMessage);
-        updateWidget();
+        _messageText = newMessage;
+        update();
     }
 }
 
-void MessageWidget::setUserName(QString newUserName)
+void MessageWidget::setUserName(const QString& newUserName)
 {
     if (newUserName != "")
     {
-        userName = newUserName;
-        updateWidget();
+        _username = newUserName;
+        update();
     }
 }
 
-void MessageWidget::setStdUserName(std::string newUserName)
+void MessageWidget::setTime(qint64 utc)
 {
-    if (newUserName != "")
-    {
-        userName = QString::fromStdString(newUserName);
-        updateWidget();
-    }
+    dateTimeMessage.setSecsSinceEpoch(utc);
+    update();
 }
 
-void MessageWidget::setDateTime(QDateTime newDataTime)
+void MessageWidget::setReactionMap(std::map<int, int> newReactionMap)
```

Comment:
here too

Reply:
this thing is now on rework and not used